app/feeds/new/add-operator/template.hbs

```diff
@@ -4,7 +4,7 @@
 
 <div class="add-operator-info">
 	<p>
-		We've fetched the GTFS feed from that URL, parsed it, and listed out the {{model.operators.length}} operator(s) listed in the feed. For each operator, please check its detail and add in any extra information about its location.
+		Transitland checked your feed and found the following number of operators: {{model.operators.length}}. Review the information below and provide additional details about each operator and location it serves. All fields are required unless otherwise indicated.
```

Comment:
This could dynamically update to say: "Transitland checked your feed and found (1 operator/X operators). Please review the information below and provide additional details about each operator and the location it serves."

Do we need to say all fields are required if we are putting (optional) after a few of the fields, or is that implied?